spec/ddtrace/contrib/rails/support/models.rb

```diff
@@ -0,0 +1,51 @@
+RSpec.shared_context 'Rails models' do
+  let(:application_record) do
+    stub_const('ApplicationRecord', Class.new(ActiveRecord::Base) do
+      self.abstract_class = true
+    end)
+  end
+
+  let(:models) do
+    # Force connection to initialize, and dump some spans
+    # TODO: Refactor this... needs it badly.
+    [].tap { application_record.connection }
+  end
+
+  # TODO: Figure out how to run database migrations to setup models
```

Comment:
As far as how to handle models/migrations, I was thinking about this a little over the weekend. My thinking was that the `models.rb` file probably isn't necessary. Instead of centralizing database migrations, I think individual specs that actually use models should be responsible for creating/migrating them in `before(:each)`/`after(:each)` blocks. Wherever we see migration code being repeated, or ActiveRecord complexities being implemented in specs, we could extract these problem spots into a helper module that can be re-composed back into specs.

For this PR, however, I'd recommend against implementing this, since it's out of scope.